Source/JavaScriptCore/API/JSRemoteInspector.cpp

```diff
@@ -41,6 +41,7 @@
 using namespace Inspector;
 
 static std::optional<bool> remoteInspectionEnabledByDefault = std::nullopt;
+static bool disablingInspectionIgnoresInternalPolicies = false;
```

Comment:
I wonder if it's a better name is without the double negative. Maybe "inspectionFollowsInternalPolicies"? And it defaults to true.

Reply:
That's a much better name for this, yes!